### ISSUE TITLE
Feature request: Instead of not allowing the user to delete the default list, don't allow deleting the last list

### ISSUE THREAD

**Issue by RedX2501** [TomBursch/kitchenowl#586]
### What's the feature 🧐

I don't see anything important about the default list.

Instead the user should just be forbidden from deleting the last shopping list.

**Comment by TomBursch** [comment]
Yeah, I guess we could do that, although the default list indicates where items are added to when you click on add to shopping list in a recipe. The "default" shopping list is just the oldest one.

**Comment by RedX2501** [comment]
In that case it probably makes sense to be able to mark one of the lists as default per household and allow each individual member to set its own default. The member default list would override the household one.
